### PR TITLE
Ephemeral accounts and cover why React Native

### LINKED ISSUE #57: Add “no platform risk” to the why section

### PR DESCRIPTION
**Pull request by pcowgill**
### Issue link

https://github.com/tasitlabs/TasitSDK/issues/57

### Auto-close the issue?

Closes #57 

### Types of changes

Docs (modified documentation or code comments)

### CODE REVIEW

**Comment by marcelomorgado** [comment]
Are you waiting for my review?

**Comment by pcowgill** [comment]
As a general rule I have merging blocked without a review (although I am able to bypass it).

So yes, I was waiting for your review. It doesn't need to be very detailed, but a quick glance couldn't hurt. Thanks!

**Comment by marcelomorgado** [comment]
> As a general rule I have merging blocked without a review (although I am able to bypass it).
> 
> So yes, I was waiting for your review. It doesn't need to be very detailed, but a quick glance couldn't hurt. Thanks!

Ok. I was in doubt because I'm not listed as a reviewer, unlike others PRs.

**Comment by pcowgill** [comment]
> Ok. I was in doubt because I'm not listed as a reviewer, unlike others PRs.

Ah okay - I'll be sure to be more intentional about that in the future. I forgot this time. Thanks!

File: README.md [hunk 1]
```diff
@@ -101,6 +109,18 @@ import Account from "tasit-account";
...
 ...with the rest of the code remaining the same.
 
+Or maybe you just want to use the simple abstraction for interacting with smart contracts. Then just install `tasit-action` directly instead.
+
+```
+npm install --save tasit-action
+```
+
+```javascript
+import Contract from "tasit-action";
```
Comment (marcelomorgado) [inline README.md]: Is that:
`import { Contract } from "tasit-action"`
the correct way?

Comment (pcowgill) [reply README.md]: It crossed my mind while typing it that I might have it wrong. Thanks for pointing this out. Also feel free to use the GitHub suggestions feature so I can insta-accept it

suggestion

Comment (marcelomorgado) [inline README.md]:
```suggestion
import { Contract } from "tasit-action";
```

Comment (marcelomorgado) [reply README.md]: Thanks.

**Pull request merged**